### PR TITLE
Feature/rpnv1 99 test logging

### PR DESCRIPTION
**Pull request by clstrfsck**
Update to Log4j2

### CODE REVIEW

**Review by Sajjon — APPROVED** [review]
LGTM

**Review by talekhinezh — APPROVED** [review]

File: radixdlt/src/main/resources/log4j2.properties [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+appender.rolling.layout.pattern = %d{ISO8601} [%p/%c{1}/%t] (%F:%L) - %m%n
+appender.rolling.policies.type = Policies
+appender.rolling.policies.time.type = TimeBasedTriggeringPolicy
+appender.rolling.policies.time.interval = 1
+appender.rolling.policies.time.modulate = true
+appender.rolling.strategy.type = DefaultRolloverStrategy
+# Two weeks = 14 days of log rollover
+appender.rolling.strategy.max = 14
+
+#appender.json.type = RollingFile
```
Comment (flotothemoon) [inline radixdlt/src/main/resources/log4j2.properties:28]: What's with all this commented out configuration stuff?

**Review by flotothemoon — APPROVED** [review]
Finally :)

File: radixdlt/src/main/resources/log4j2.properties (hunk 1 above)
Comment (clstrfsck) [reply radixdlt/src/main/resources/log4j2.properties:28]: It's there in order to make it easy to include if you want it.  Not going to turn it on by default as it consumes a lot of I/O, and CPU to generate and store.

**Comment by clstrfsck** [comment]
Going to merge anyway, despite flakey unit tests on travis.

**Pull request merged**